syntaxes/Major.tmLanguage

```diff
@@ -315,7 +315,7 @@
 			<key>comment</key>
 			<string>control-transfer-statement</string>
 			<key>match</key>
-			<string>\b(continue|break|return)\b</string>
+			<string>\b(continue|break|return|assert:invariant|assert:func|assert:return|assert:\d+:async)\b</string>
```

Comment:
This PR is currently using the "control.transfer" token highlighting (since it looks different in the default VS Code themes), but we might be able to split this back into its own pattern.